src/web/app/components/course-edit-form/course-edit-form.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="card">
+<div>
```

Comment:
I'd rather we keep the `card` class here because this div's children elements are using `card-header` and `card-body`, so logically the div should be a `card`.

The issue of the excess margin actually comes from the `instructor-course-edit-page` component - if you take a look at line 2, the `card` class is being applied to the div surrounding the `course-edit-form` component, hence the extra padding. Simply removing the class there should resolve the issue, similar to how the `instructor-edit-panel` component is used (lines 19-33)

Reply:
Is it right now? and what to do with the failed test cases? Thank you.